nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/main.css

```diff
@@ -59,10 +59,6 @@ div.context-menu-provenance {
     text-overflow: ellipsis;
 }
 
-.hide-multi-line {
-    max-height: 16px;
-}
-
```

Comment:
Note: No longer needed now that we're using the `multi-line-clamp-ellipsis` class.